Framework/CCDBSupport/src/CCDBHelpers.cxx

```diff
@@ -217,7 +218,9 @@ auto populateCacheWith(std::shared_ptr<CCDBFetcherHelper> const& helper,
     const auto url2uuid = helper->mapURL2UUID.find(path);
     if (url2uuid != helper->mapURL2UUID.end()) {
       etag = url2uuid->second.etag;
-      checkValidity = std::abs(int(timingInfo.tfCounter - url2uuid->second.lastCheckedTF)) >= chRate;
+      // We check validity every chRate timeslices or if the cache is expired
+      uint64_t validUntil = url2uuid->second.cacheValidUntil;
+      checkValidity = (std::abs(int(timingInfo.tfCounter - url2uuid->second.lastCheckedTF)) >= chRate) || (validUntil <= timestamp);
```

Comment:
Should not it be `&& (validUntil <= timestamp)` ?

Reply:
@ktf my yesterday fix was not concerning this `||` but line before it: https://github.com/AliceO2Group/AliceO2/pull/12885/files